flux-kube/lib/kube_translator.py

```diff
@@ -0,0 +1,82 @@
+##############################################################
+# Copyright 2021 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+##############################################################
+
+import warnings
```

Comment:
Out of curiosity. Have you apply Python black for format checks? (You may have already done that, in which case sorry for the noise). 

https://github.com/flux-framework/flux-core/blob/7c42cecc7e5f4998b73ecaa53d67f0a3da0156da/.github/workflows/main.yml#L27

Reply:
I just tried it, and it helped a lot in some cases and seems to have rendered other lines unreadable. I may cherry pick some changes it made.